lib/src/notifications/notification_service.dart

```diff
@@ -34,6 +31,14 @@ class NotificationService {
     }
   }
 
+  final StreamController<PlayerControlCommand> _commandController =
+      StreamController<PlayerControlCommand>.broadcast();
+
+  /// Stream of remote player command send by native side
```

Comment:
Not sure I fully understood this sentence, but it should be in the past I think
```suggestion
  /// Stream of remote player command sent by native side
```